web/unit_tests/cases/element_interfaces.js

```diff
@@ -150,25 +150,37 @@ if(typeof InterfaceTests == 'undefined') {
 
       var node = pair.elem.childNodes[0];
       var sel = document.getSelection();
-      if(node.nodeType == 3) {
-        if(device.browser == 'ie') {
-          if(start > end) {
-            // The Range API doesn't allow 'backward' configurations.
-            var temp = end;
-            end = start;
-            start = temp;
-          }
 
-          var range = document.createRange();
-          range.setStart(node, start);
-          range.setEnd(node, end);
+      var setIESelection = function(node, sel, start, end) {
+        if(start > end) {
+          // The Range API doesn't allow 'backward' configurations.
+          var temp = end;
+          end = start;
+          start = temp;
+        }
+
+        var range = document.createRange();
+        range.setStart(node, start);
+        range.setEnd(node, end);
 
-          sel.removeAllRanges();
-          sel.addRange(range);
+        sel.removeAllRanges();
+        sel.addRange(range);
+      }
+
+      if(node.nodeType == 3) {
+        if(device.browser == 'ie') {
+          setIESelection(node, sel, start, end);
         } else {
+          sel.removeAllRanges();
           // Does not work on IE!
-          sel.setPosition(node, start);
-          sel.extend(node, end);
+          try {
+            sel.setPosition(node, start);
+            sel.extend(node, end);
+          } catch (e) {
```

Comment:
Addresses a weird testing-setup error that's affecting Firefox when run through BrowserStack.  It's only the one browser, so at least the affected cases (reverse-selection correctness) are well-covered by Chrome, Edge, and Safari.